### PR TITLE
Makes buttcrabs immune to tox damage

### LINKED ISSUE #10910: butt spiders can die from their own farts
### Labels

[BUG][MINOR]

### Description

same as title

### Steps to reproduce

1. be buttspider
2. fart (10% chance for very toxic farts must happen)
3. inahle the gas cloud
4. fucking die

### Expected Behavior

not inhaling it

### Additional Information & Screenshots

Reported by: ZWRh3
On server: Goonstation 1 Classic: Heisenbee
Round log date: 2022-09-20-06-09
Reported on: 2022-09-20 07:13:29

### PR DESCRIPTION
**Pull request by Debi-sys**
## About the PR 
It makes buttcrabs immune to tox damage.

## Why's this needed? 
It makes no sense for buttcrabs to be able to take tox damage as they are a sentient ass with legs. It also fixes #10910 

## Changelog

```changelog
(u)golferjoe
(+)Buttcrabs are now immune to tox damage.
```

### CODE REVIEW

**Comment by DimWhat** [comment]
Finally, a butt crab buff

File: code/mob/living/critter/changeling_critters.dm [hunk 1]
```diff
@@ -257,7 +257,6 @@
 	setup_healths()
 		add_hh_flesh(5, 1)
 		add_hh_flesh_burn(4, 1.25)
-		add_health_holder(/datum/healthHolder/toxin)
```
Comment (Tarmunora) [inline code/mob/living/critter/changeling_critters.dm:260]: this is handspiders though?

Comment (Debi-sys) [reply code/mob/living/critter/changeling_critters.dm:260]: Oh my god i didn't notice, I'll fix this when I'm home. Thank you for noticing

Comment (Debi-sys) [reply code/mob/living/critter/changeling_critters.dm]: Yup fixed that now, safe to merge i think.

**Pull request merged**